components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/ExceptionCodes.java

```diff
@@ -192,6 +192,11 @@
     MULTIPLE_USERS_EXIST(900609, "Multiple users with the same username exist in the system", 500, "Multiple " +
             "users with the same username exist in the system"),
     INVALID_USER_ROLES(900610, "Invalid user roles found", 400, "Invalid user roles found"),
+    IDP_ADDING_FAILED(900611, "Unable to add identity provider", 400, "Error while adding the identity provider"),
```

Comment:
an*

Reply:
> When adding db changes, we have to check how we propagate these changes to existing deployments. We cannot have any downtime. I guess you are handling these concerns as well.

I created a doc issue for (master docs) [1] to update the migration scripts.

We can use the same SQL statements for Choreo DB as well. @sumedhassk and @vlgunarathne any idea?

[1] https://github.com/wso2/docs-apim/issues/4163

Reply:
Fixed via 7d86bcf96daa8bbd55e9b701d7d49b6225cbbb75